### PR TITLE
Modify compaction testcase

### LINKED ISSUE #19324: [Bug]: [Nightly] Some cases related to compaction failed
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:5141e05
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev30
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

[2022-09-20T21:08:05.850Z] [gw2] [ 11%] FAILED testcases/test_compaction.py::TestCompactionOperation::test_compact_less_threshold_no_merge
[2022-09-20T21:08:30.494Z] [gw1] [ 12%] FAILED testcases/test_compaction.py::TestCompactionOperation::test_compact_no_merge
[2022-09-20T21:10:38.016Z] [gw4] [ 15%] FAILED testcases/test_compaction.py::TestCompactionParams::test_compact_max_time_interval

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Milvus Log

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by NicoYuan1986**
Resolve: #19324
Signed-off-by: “nico” <cheng.yuan@zilliz.com>

### CODE REVIEW

File: tests/python_client/testcases/test_compaction.py [hunk 1]
```diff
@@ -322,6 +322,10 @@ def test_compact_max_time_interval(self):
 
         sleep(ct.max_compaction_interval + 1)
 
+        # create index
+        default_index = {"index_type": "IVF_FLAT", "params": {"nlist": 128}, "metric_type": "L2"}
+        collection_w.create_index("float_vector", default_index)
+
         # verify queryNode load the compacted segments
```
Comment (ThreadDao) [inline tests/python_client/testcases/test_compaction.py:329]: Thank you so much for the update it's great. Two little tips:
1. Replace `float_vector` with var `ct.default_float_vec_field_name`
2. No need to create a new variable `default_index`, just use `ct.default_index` directly

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_compaction.py:329]: > Thank you so much for the update it's great. Two little tips:
> 
> 1. Replace `float_vector` with var `ct.default_float_vec_field_name`
> 2. No need to create a new variable `default_index`, just use `ct.default_index` directly

OK!

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_compaction.py:328]: > Thank you so much for the update it's great. Two little tips:
> 
> 1. Replace `float_vector` with var `ct.default_float_vec_field_name`
> 2. No need to create a new variable `default_index`, just use `ct.default_index` directly

Done.Thanks for your advice. ^_^

**Pull request merged**